File: app/src/main/java/com/shuyu/apprecycler/chat/data/model/ChatTextModel.java
package com.shuyu.apprecycler.chat.data.model;

import java.io.Serializable;

/**
 * 文本model
 * Created by guoshuyu on 2017/9/4.
 */

public class ChatTextModel extends ChatBaseModel implements Serializable {

    private String content;

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: app-kotlin/src/main/java/example/shuyu/recycler/kotlin/chat/data/model/ChatTextModel.kt
package example.shuyu.recycler.kotlin.chat.data.model

import java.io.Serializable

/**
 * 文本model
 * Created by guoshuyu on 2017/9/4.
 */

open class ChatTextModel : ChatBaseModel(), Serializable {

    var content: String? = null
}
